### DISCUSSION TITLE
How to remove a private key from a harvester machine

### DISCUSSION THREAD

**Discussion by westfruit** [Chia-Network/chia-blockchain#6737] [Support]
I have  several harvester on different server but i execute chia add key command some time on that. now for security i want remove the private key from harvesters and not affect havester working ,how can i do?

**Comment by loppefaaret** [comment]
if you execute `chia keys delete_all` it will remove all keys on the machine - please ensure you have the keys 24 word mnemonic backed up before doing so!

**Comment by BrandtH22** [comment]
@westfruit , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
